tests/cases/standalone/common/function/time.sql

```diff
@@ -5,3 +5,5 @@ select current_time();
 select GREATEST('1999-01-30', '2023-03-01');
 
 select GREATEST('2000-02-11'::Date, '2020-12-30'::Date);
+
+SELECT to_timezone('2022-09-20T14:16:43.012345+08:00'::Timestamp, 'Europe/Berlin');
```

Comment:
nit:
1) I've noticed inconsistency in the SQL format (upper/lower case). Not a big deal, just decided to call out
2) maybe a test for the failing scenario (with unknown timezone for example)

Reply:
> upper/lower case

What does it mean? What's the SQL format different from this?